conans/test/functional/build_helpers/cmake_flags_test.py

```diff
@@ -475,8 +475,10 @@ def package(self):
         client.run("create . danimtb/testing")
         if platform.system() == "Windows":
             self.assertIn("WARN: CMake generator could not be deduced from settings", client.out)
-            self.assertIn('Configure command: -DCONAN_EXPORTED="1" -DCONAN_IN_LOCAL_CACHE="ON" '
-                          '-DCMAKE_INSTALL_PREFIX=', client.out)
+            self.assertIn('Configure command: -DCONAN_EXPORTED="1" '
+                          '-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY="ON" -DCMAKE_INSTALL_PREFIX=',
```

Comment:
This means that now we don't check for IN_LOCAL_CACHE=ON? Sounds something could break.